src/main/resources/assets/securitycraft/lang/ru_ru.json

```diff
@@ -780,7 +782,6 @@
     "gui.securitycraft:rift_stabilizer.moduleRequired": "Вам нужно установить Умный Модуль, чтобы переключать, какие типы телепортации могут быть обнаружены этим Стабилизатором Разлома.",
     "gui.securitycraft:rift_stabilizer.teleportationTypes": "Обнаруживаемые Типы Телепортации:",
     "gui.securitycraft:rift_stabilizer.toggle": "Нажмите на записи в списке ниже, чтобы переключить обнаружение этого конкретного типа телепортации.",
-    "gui.securitycraft:scManual.author": "Переведено cutiegin",
```

Comment:
This line has changed position, which is not correct. We are sorting our lines alphabetically by key (the "gui.securitycraft..." bit), so it should stay where it was. You can easily sort the lines either in Notepad++ by going to Edit > Line Operations > Sort Lines Lexicographically Ascending, or by running the sort_language_files.py python script that is found in the main folder of this repository.

Reply:
Yup. You need to have [python ](https://www.python.org/downloads/release/python-3124/) installed, though.